cms/static/js/views/video/translations_editor.js

```diff
@@ -89,7 +89,7 @@ function($, _, AbstractEditor, FileUpload, UploadDialog) {
                 items = this.$el.find('ol').find('.list-settings-item');
 
             _.each(items, function(element, index) {
-                var key = $(element).find('select').val(),
+                var key = $(element).find('select option:selected').val(),
```

Comment:
Perhaps this was before jquery-migrate was properly installed? I agree though that the new syntax is  the correct one.